search-app/src/cmr/search/results_handlers/atom_json_results_handler.clj

```diff
@@ -73,6 +75,7 @@
                        :has_granules (when has-granules-map (or (< 0 granule-count)
                                                                 (get has-granules-map id false)))
                        :granule_count (when granule-counts-map granule-count)
+                       :has-variables has-variables
```

Comment:
Again, not sure of the reason for the difference between under and dash here?

Reply:
Historically ECHO used underscores in all APIs. When transitioning to CMR a decision was made that all APIs should continue to use underscores rather than dashes.

Reply:
Good catch. Updated.